ui/js/models/resultset.js

```diff
@@ -191,39 +191,39 @@ treeherder.factory('ThResultSetModel', ['$rootScope', '$http', '$location',
                 return $http.post(thUrl.getProjectUrl("/resultset/", repoName) + uri);
             },
 
-            triggerMissingJobs: function (resultset_id, repoName, decisionTaskID) {
+            triggerMissingJobs: function (resultset_id, repoName, decisionTaskId) {
                 var uri = resultset_id + '/trigger_missing_jobs/';
                 return $http.post(thUrl.getProjectUrl("/resultset/", repoName) + uri).then(function () {
-                    return tcactions.load(decisionTaskID).then((results) => {
+                    return tcactions.load(decisionTaskId).then((results) => {
```

Comment:
This was actually different in a lot of places before this set of commits for a long time. I just decided to fix it up in this one because otherwise the nice object assignment stuff would've been ugly.